calculator_bronze.rb

```diff
@@ -0,0 +1,42 @@
+puts "Hello! Welcome to my calculator!\nWhat kind of math operation would you like to use?"
```

Comment:
Cool! Nice use of the line break \n. Consider adding to the puts statement so the user knows what options to choose from and know what to type in (+, -, *, etc).